src/dataflow-types/src/client/controller/compute.rs

```diff
@@ -312,32 +314,45 @@ impl<'a, C: Client<T>, T: Timestamp + Lattice> ComputeController<'a, C, T> {
 
         // // Validate that the ids exist.
         // self.validate_ids(frontiers.iter().map(|(id, _)| *id))?;
+        let policies = frontiers
```

Comment:
I know this is existing code, but is there a reason that the ID validation one line up is disabled?

Reply:
It crashes on `main` when it is enabled (until https://github.com/MaterializeInc/materialize/pull/10835 lands).